deepmd/dpmodel/descriptor/repformers.py

```diff
@@ -401,6 +401,10 @@ def has_message_passing(self) -> bool:
         """Returns whether the descriptor block has message passing."""
         return True
 
+    def need_sorted_nlist_for_lower(self) -> bool:
+        """Returns whether the descriptor block needs sorted nlist when using `forward_lower`."""
+        return True
```

Comment:
Shouldn't be `False`? Suppose the `repformer` is used alone it does not need a sorted nlist.

Reply:
I see, exactly. Fixed in https://github.com/deepmodeling/deepmd-kit/pull/3993/commits/be56b5e431c13ea9cff32d1cd98933849bfea9c3 .